engine/dlib/src/dlib/message.cpp

```diff
@@ -354,12 +354,48 @@ namespace dmMessage
         return false;
     }
 
-    void ResetURL(const URL& url)
+    void ResetURL(URL* url)
     {
-        memset((void*)&url, 0, sizeof(URL));
+        memset((void*)url, 0, sizeof(URL));
```

Comment:
1) memset'ing a "const" is a bit confusing
2) Using a pointer/handle is usually our preferred way. I opted to change that now that the api goes public.